ash/src/entry.rs

```diff
@@ -46,6 +44,22 @@ pub enum LoadingError {
     LibraryLoadError(String),
 }
 
+impl fmt::Display for LoadingError {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "LoadingError::{:?}", self)
+    }
+}
+
+impl Error for LoadingError {
+    fn description(&self) -> &str {
+        "LoadingError"
+    }
```

Comment:
This method is deprecated; we shouldn't implement it explicitly.

Reply:
I was following the precedent set with the other error types.

Reply:
You can update those to current conventions if you like, but either way we shouldn't be introducing new code using obsolete patterns.